### PR TITLE
[docs] FAQ: How do I customize messages on Novu?

### PR DESCRIPTION
**Pull request by Akshu-on-github**
- **What kind of change does this PR introduce?** Added an FAQ - "How do I customize messages on Novu?"

- **Why was this change needed?** I feel like this is one of the first questions that a new user would ask

- **Other information**: NA

### CODE REVIEW

**Review by florian-lefebvre — COMMENTED** [review]
Hey looking good! Could you provide a screenshot?

**Comment by Akshu-on-github** [comment]
Hey @florian-lefebvre,
Could you please elaborate on what I should take a screenshot of?

**Comment by florian-lefebvre** [comment]
Just to make sure if the table looks fine in the FAQ layout

File: docs/docs/community/faq.md [hunk 1]
```diff
@@ -10,3 +10,29 @@ Novu consists of multiple services written in Node.js and Typescript. The follow
...
+  ```
+
+- **Iteration/Looping:** You can iterate through a list of variables using the [each](https://docs.novu.co/platform/templates#iteration) keyword. The code within the block will loop once for each element that is present in the iterable variable. The syntax is as follows:
+  ```
+  {{#each iterable}}
+    ...
+  {{/each}}
+  ```
+
+- **Conditional Block**
```
Comment (ainouzgali) [inline docs/docs/community/faq.md:30]:
```suggestion
- **Conditional Block:**
```

File: docs/docs/community/faq.md [hunk 2]
```diff
@@ -10,3 +10,29 @@ Novu consists of multiple services written in Node.js and Typescript. The follow
 - MongoDB
 - Redis
 - File storage (S3/GCP/Azure) - Optional
+
+## How do I customize messages on Novu?
+
+In Novu, [handlebar variables](https://handlebarsjs.com/guide/) (variables enclosed within double curly brackets) are used to customize messages. Using this, one can take advantage of the following tools:
+
+- **Custom Variables:** You can create [custom variables](https://docs.novu.co/platform/templates#variable-usage) by defining it in the payload and access the value of the same using the following syntax:
```
Comment (ainouzgali) [inline docs/docs/community/faq.md:18]:
```suggestion
- **Custom Variables:** You can create [custom variables](https://docs.novu.co/platform/templates#variable-usage) by defining it in the payload and accessing its value using the following syntax:
```

What do you think? Something feels off to me in the sentence 'the value of the same'🤔

**Review by ainouzgali — COMMENTED** [review]
Hi @Akshu-on-github thank you so much for contributing🤩 Very much appreciated! 
Let me know what you think of my suggestions😊

File: docs/docs/community/faq.md (hunk 2 above)
Comment (Akshu-on-github) [reply docs/docs/community/faq.md:18]: Yeah, this sounds a lot more natural! ✅

**Pull request merged**